include/swift/SIL/SILLocation.h

```diff
@@ -69,8 +69,6 @@ class SILLocation {
 
 public:
   enum LocationKind : unsigned {
-    // FIXME: NoneKind is to be removed.
-    NoneKind = 0,
     RegularKind = 1,
     ReturnKind = 2,
     ImplicitReturnKind = 3,
```

Comment:
At this point I would just let RegularKind start at zero.
Also, can you make sure the validation tests pass? I could imagine this being accidentally during by SIL deserialization.